docs/http_access_API_definition.md

```diff
@@ -740,7 +740,7 @@ __Request__
 |---|---|---|---|
 |groupName|yes|the group name to set flow control rule|String|
 |enableBind|no|whether to bind consuming permission, default value 0 means disable|int|
-|allowBCleintRatio|no|the ratio of the number of the consuming target's broker against the number of client in consuming group|int|
+|allowBClientRatio|no|the ratio of the number of the consuming target's broker against the number of client in consuming group|int|
```

Comment:
This needs to be modified together with the external api parameters and configuration parameters, or will the document be inconsistent with the actual field

But after this modification, it will affect the already used business adjustments. How can it be compatible?